src/Microsoft.ML.AdditionalLearners/Microsoft.ML.AdditionalLearners.csproj

```diff
@@ -0,0 +1,17 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netstandard2.0</TargetFramework>
+    <IncludeInPackage>Microsoft.ML.AdditionalLearners</IncludeInPackage>
+    <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
```

Comment:
I don't think `AllowUnsafeBlocks` is necessary.  I couldn't find `unsafe` in the `OlsLinearRegression.cs` file.